packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -39,13 +39,14 @@ export const processPatientDocumentRequest = async (
   patientId: string,
   whType: MAPIWebhookType,
   status: MAPIWebhookStatus,
-  documents?: DocumentReferenceDTO[]
+  documents?: DocumentReferenceDTO[],
+  cxPayload?: object
 ): Promise<void> => {
   try {
     const settings = await getSettingsOrFail({ id: cxId });
     // create a representation of this request and store on the DB
     const payload: WebhookPatientDataPayloadWithoutMessageId = {
-      patients: [{ patientId, documents, status }],
+      patients: [{ patientId, documents, status, cxPayload }],
```

Comment:
I think this request meta should be a top-level object. Just gotta figure out a good name, since we already use `meta` for our own metadata. Maybe something like `documentQueryMeta`?